libtiff/bitarray-a1646c0/bitarray/test_bitarray.py

```diff
@@ -14,11 +13,12 @@
 
 if is_py3k:
     from io import StringIO
+    import unittest as ut
 else:
     from io import StringIO
+    import unittest2 as ut
```

Comment:
So `unittest2` is a Python 2.6 thing. Not a Python 2 thing. IOW `unittest` in Python 2.7 is the same as `unittest`, but already included. Could this check be updated to reflect this please?